README.md

```
@@ -17,6 +17,29 @@ $ npm install @babel/runtime
+- Forks the RSK Mainnet network (`https://public-node.rsk.co`)
+- Sets the chain ID to 30 (RSK Mainnet)
+- Runs the local node on port 8545 (default)
+
+Once Anvil is running, you can run the tests in a separate terminal:
+
+```bash
+yarn test
+```
+
+> **Note**: Make sure Anvil is running before executing tests, as the test suite requires a local blockchain node to be available at `http://127.0.0.1:8545`.
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)